src/pages/Admin/Users/index.jsx

```diff
@@ -236,7 +236,7 @@ export default class Users extends Component {
       selectName
     } = userStore;
 
-    const data = userStore.users.toJSON();
+    const data = toJS(userStore.users);
```

Comment:
`toJS` is deprecated in mobx, recommend to use `toJSON`

Reply:
If use toJSON,  modify user name will also change table list data. Becase observable lead to data bidirectional binding.